src/components/picker/index.js

```diff
@@ -224,19 +224,9 @@ class Picker extends PureComponent {
     };
   };
 
-  shouldNotChangePickerLabelWhileSelecting = () => {
-    const {mode} = this.props;
-    return mode === Picker.modes.MULTI;
-  };
-
   getLabelValueText = () => {
-    const {value: propsValue} = this.props;
-    const {value: stateValue} = this.props;
-
-    if (this.shouldNotChangePickerLabelWhileSelecting()) {
-      return this.getLabel(propsValue);
-    }
-    return this.getLabel(stateValue);
+    const {value} = this.props;
+    return this.getLabel(value);
   };
```

Comment:
I think that now getLabelValueText is redundant.. 
it's just a wrapper of `this.getLabel`. 
Can we remove it?

Reply:
Right 😁

Reply:
I think it was valid before when you had some logic there.. 
But now we use it only in one place and all it does it extract the value prop. Which seems a little redundant, no?